### ISSUE TITLE
 Search in rails admin not working on associated models

### ISSUE THREAD

**Issue by aravindgdf22** [sferik/rails_admin#2359]
When I do a search on the table the query only works on email not on username and full name of the user and guest user tables which is basically an associated model for the invited email model.
```
config.model InvitedEmail do
  list do
    field :id do
      searchable false
    end
    field :email do
      searchable :email
    end 
    field :user do
      searchable [:username, :full_name]
    end
    field :guest_user do
      searchable [:username, :full_name]
    end
  end
end
```
Here is the log on searching in the list:
```
Parameters: {"model_name"=>"invited_email", "utf8"=>"✓", "query"=>"asd", "_pjax"=>"[data-pjax-container]"} (pid:54610)
InvitedEmail Load (4.4ms)  SELECT  `invited_emails`.* FROM `invited_emails` WHERE ((LOWER(invited_emails.email) LIKE '%asd%'))  ORDER BY invited_emails.id desc LIMIT 20 OFFSET 0 (pid:54610)
```

**Comment by fmh** [comment]
Main topic #1434 and same issue #2346

**Comment by mshibuya** [comment]
Fixed by b92a4d1a30b706d08df2aee4d0a59dad698a0552.
